### DISCUSSION TITLE
did KB5007215 break the explorer patcher?

### DISCUSSION THREAD

**Discussion by anub1s22** [valinet/ExplorerPatcher#323] [Q&A]
first of all thank you for this patch it's made windows 11 a LOT more comfortable to use.

but yesterday i installed KB5007215 (set it to install when i went off to bed) and this morning my clock was missing on the taskbar  and i couldn't get it back, so i uninstalled the version i had (renamed the file to DXGI.dll.old and rebooted) which restored the windows 11 task bar and i grabbed the most recent version of the ExplorerPatcher installed it the way i installed the old one (place in C:/windows and reboot) but now my task bar is just an opaque slab with only the tray icons on it...though i did not do that current step where you kill the process and it downloads some stuff so i guess that might be it.

correction i just did that step and ehr...windows was not amused the explorer kept on restarting indefinitely and the taskbar was still a slab with tray icons. upon a reboot...i did not get in to windows, upon another reboot my entire desktop was the color of the slab. and the 3rd one my screen was black, after the 4th boot it triggered the startup repair so i could hop in to safe mode...which...also had no task bar to speak of and a only black screen but i could CTRL+ALT+DEL to get to the taskmanager, from there i could open a process's file location and from there i could slap .old on to dxgi.dll...yea something seems borked >.>

**Comment by valinet** [comment]
> first of all thank you for this patch it's made windows 11 a LOT more comfortable to use.

You're welcome.

For me, it works just fine on the latest CU. Idk, you have to debug the issues for yourself, as usual, make sure there are no additional programs that interfere etc: read about the experience another user reported here, see if the pointers in that thread help you get a solution: https://github.com/valinet/ExplorerPatcher/issues/320

Thanks

**Comment by anub1s22** [comment]
ah digging through computer management's event viewer i've located the same explorer.exe crashing issue with the same details...but not the application causing it....and no idea what might be interfering otherwise so i'll see about getting a dump generated and hopefully figure out what's messing with it, maybe tomorrow when i have some more time uninstall the update and see what happens (as far as i can tell the update does barely anything so i find it odd that it wrecked my clock...and seemingly more)

**Reply by valinet** [reply]
Good luck, hopefully you get it solved.
